src/uwtools/drivers/experiment.py

```diff
@@ -36,6 +36,9 @@ def validate_config(self, config_file):
         """
         Validate the configuration file.
         """
+        config_validation = getattr(import_module(self.modname), "validate_config")  # pragma: no cover
```

Comment:
I saw that CI failed (https://github.com/ufs-community/workflow-tools/actions/runs/5616367277/job/15218559714?pr=268) due to this line being too long. `make format` will take care of that for you, along with most other formatting concerns. Always a useful thing to run (and any potential changes it makes committed) before a `git push`. My habit is to do `make format && make test` before a push (and pretty often anyway).

Reply:
Thanks - I 100% forgot to do that, trying to see if I can add to my VS Code tutorial with support for this but its a little fussy.